articles/aks/cluster-autoscaler-overview.md

```diff
@@ -43,7 +43,8 @@ It's a common practice to enable cluster autoscaler for nodes and either the Ver
   * A pod uses node selectors or anti-affinity that can't be honored if scheduled on a different node.
     For more information, see [What types of pods can prevent the cluster autoscaler from removing a node?](https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/FAQ.md#what-types-of-pods-can-prevent-ca-from-removing-a-node).
 >[!IMPORTANT]
-> **Do not make changes to individual nodes within the autoscaled node pools**. All nodes in the same node group should have uniform capacity, labels, taints and system pods running on them. 
+> **Do not make changes to individual nodes within the autoscaled node pools**. All nodes in the same node group should have uniform capacity, labels, taints and system pods running on them.
+* The cluster autoscaler is not responsible for enforcing a "maximum node count" in a cluster node pool irrespective of pod scheduling considerations. If any non-cluster autoscaler actor sets the node pool count to a number beyond the cluster autoscaler's configured maximum, cluster autoscaler will not automatically remove nodes. Scale down behaviors of cluster autoscaler remain scoped to removing only nodes that have no scheduled pods. The sole purpose of the cluster autoscaler's max node count configuration is to enforce an upper limit for scale up operations. It does not have any effect upon scale down considerations.
```

Comment:
@jackfrancis - do you think this would sound more prescriptive 

"The Cluster Autoscaler does not enforce a "maximum node count" in a node pool based on pod scheduling. When adjusting the maximum node count of a node pool, ensure it is equal to or greater than the current node count in the nodepool. The Cluster Autosaler scales down by removing nodes only when they have no scheduled pods. The max node count setting exists solely to enforce an upper limit for scale-up operations and does not affect scale-down decisions."

Reply:
I think it sounds like a reworded version of the same concept, but certainly could be an improvement.

@tallaxes @Bryce-Soghigian @rakechill which one do we prefer?